Tower hardware simulations

vbogdev opened the pull request:
Set simulations to correct time, added fact that 12 AWG wire is to be used, updated voltage chart for lead acid battery, added summary of other battery types.

vbogdev committed:
Updated simulation report, added heat sink

Includes calculations for heat sink and list of parts.

vbogdev committed:
added gitignore

vbogdev committed:
updated report and simulation

ntlhui commented on tower_battery/simulations/simulations_report.md:
```diff
@@ -0,0 +1,37 @@
+<h1>Thermal Issues</h1>
+The information below is just a brief overview of heatsinks, although electrical enclosures will also have to be used. To avoid the high temperatures of the diode melting the plastic wire connectors or the 12 AWG cable, a heat sink is needed for the diode. I recommend [this one](https://www.cuidevices.com/product/resource/hss-b20-np-12.pdf) , as it is quite cheap and is able to absorb quite a lot ...
+
+<h1>Alternative battery chemistries</h1>
+
+ - NiMH: Low energy storage, most of these batteries do not go above 20AH, would be very expensive to use many batteries in parallel. 
+
+ - Litium Ion: Very expensive, can be dangerous if allowed to overheat, aside from this is a viable alternative, but considering diode heating issues, not the best option.
+
+ - LiCoO2: Short lifespan, low thermal stability, not a good fit.
+
+ - LiFePO4: Same weight as lead acid batteries (about 20 pounds for at least 33 hours), no issues with needing to be at above 50% charge. Has a more significant voltage change as the battery discharges (4V difference between 100% and 0%), but this should still not be dangerous with diodes. Best option other than lead acid batteries.
+
+ - LiNiMnCoO2:Does not have both high energy storage and high energy output, might be difficult to find a good fitting battery of this kind.
```
Do you have sources/numbers to cite for these?

vbogdev replied:
Found a new table that was much more informative than my previous source and added links to table in document. NiCds would also probably work pretty well, especially for their cost, so I was wrong with what I said during the hardware standup from friday. Also updated simulations, inrush current issue was because of large timestep, actual jump in current is much more brief.

vbogdev committed:
Added source for chemistries, updated battery chemistries.

vbogdev committed:
Update simulations_report.md

Added parts lists for diode test, added cost estimations for chemistries and comparisons for batteries.

vbogdev committed:
Update simulations_report.md
